internal/lookout/ui/src/containers/lookoutV2/JobsTableContainer.tsx

```diff
@@ -822,7 +822,7 @@ export const JobsTableContainer = ({
             // is exactly equal to `pageSize`, then this is going to produce a
             // misleading description (e.g., "1-50 of more than 50", even though
             // there are exactly 50 rows).
-            count={data.length < pageSize ? data.length : -1}
+            count={data.length < pageSize ? pageIndex * pageSize + data.length : -1}
```

Comment:
This is the main change: I forgot to add the offset when computing the total row count.